workbench-core/audit/src/baseExtractor.ts

```diff
@@ -33,10 +33,14 @@ export const BaseExtractor: Extractor = {
       };
 
       // Sets actor to be the AuthenticatedUser's id.
-      const authenticatedUser: AuthenticatedUser = retrieveUser(res);
-      const actor = {
-        uid: authenticatedUser.id
-      };
+      let actor = { uid: 'user not found' };
```

Comment:
@htztomic are we ok with using this approach? With this approach we can still log auth requests like `/login` and `/logout`. This allows the auditing logs to be more complete since all requests will still be logged. Without this change, when the `BaseExtractor` tries to `retrieveUser()` it'll error out and the API request won't complete.

Reply:
will approve, but cut a ticket for @htztomic to review when he returns. Looks like it should be good to me.

Reply:
Thanks David. I'll put the ticket here as reference for the future. `MAFoundation-545`